clusters/app.ci/pr-reminder/pr-reminder-config.yaml

```diff
@@ -64,3 +64,10 @@ data:
           - devconsoleui
         repos:
           - openshift/console
+      - teamMembers:
+          - cybertron
```

Comment:
These should be `kerberos` id's rather than gh usernames. A presubmit is supposed to verify this, but it is not working correctly.

Reply:
Thanks. Updated it